flink/interpreter/src/main/java/org/apache/zeppelin/flink/TableEnvFactory.java

```diff
@@ -55,30 +55,50 @@
   private Executor executor;
   private org.apache.flink.api.scala.ExecutionEnvironment benv;
   private org.apache.flink.streaming.api.scala.StreamExecutionEnvironment senv;
-  private TableConfig tblConfig;
+  private TableConfig sTableConfig;
+  private TableConfig bTableConfig;
```

Comment:
It will be easy to understand if there is an explanation about the difference between "stable" and "btable".